properties.schema

```diff
@@ -101,31 +101,31 @@
       "help": "What number the scale should end on"
     },
     "_correctAnswer": {
-      "type":"number",
-      "default": 1,
+      "type": "string",
```

Comment:
Yes, it will, but it's per the README.  The problem is that the component appears to have been designed in a way such that if `_correctAnswer` is empty then the range kicks in.  We could probably parse it in the component itself to mitigate against non-number strings in the JSON.